pkg/apis/configuration/v1alpha1/types.go

```diff
@@ -64,9 +64,7 @@ type SelectorField struct {
 
 // TracingSpec is the spec object in ConfigurationSpec
 type TracingSpec struct {
-	Enabled      bool `json:"enabled"`
-	IncludeBody  bool `json:"includeBody"`
-	ExpandParams bool `json:"expandParams"`
+	SamplingRate string `json:"samplingRate"`
```

Comment:
did you regen the crd clients?

Reply:
> did you regen the crd clients?

No, that is pending, I mentioned in the PR description. I tried regenerating and script succeeded , as it did not complain but there are no files update actually happening. So I am checking and might need help.

Reply:
as discussed with @youngbupark  , TracingSpec struct is deep copied and not through memberwise copy, so crd update is not needed.